FAQ.md

```diff
@@ -169,22 +165,30 @@ emmake make
 emmake make install
 ```
 
-Finally, compile Essentia for Emscripten.
-```
+- Finally, compile Essentia with Emscripten as an LLVM target which can be further used for linking with your application code.
+  
+```bash
 cd path/to/essentia
+# for using KISS FFT
+emconfigure sh -c './waf configure --prefix=$EMSCRIPTEN/system/local/ --lightweight=KISS --emscripten'
+# OR
+# for using FFTW
 emconfigure sh -c './waf configure --prefix=$EMSCRIPTEN/system/local/ --lightweight=fftw --emscripten'
```

Comment:
As you are on it now, can you also update "fftw" to "FFTW" in `--lightweight` for style consistency?

Reply:
done